### ISSUE TITLE
Number of Items in the quickslots

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#272]
```
Hi,

it would be great if the number of items in the quickslots is shown as a small 
number in the quickslot.

I often have to search in the inventary to see how many potions I have left.

Greetings, Jenns
```

Original issue reported on code.google.com by `JennsR...@googlemail.com` on 9 Nov 2011 at 10:45

**Comment by GoogleCodeExporter** [comment]
```
I strongly support the idea.
```

Original comment by `SamuelPl...@gmail.com` on 9 Nov 2011 at 3:01

**Comment by GoogleCodeExporter** [comment]
```
I'd like this as well.  I'd also like to be able to position them on the 
screen.  For instance, instead of being on the left hand side stacked 
vertically in landscape mode and centred on the bottom in portrait mode, I 
would like them on stacked vertically on the right hand side with both.  If I 
were left handed, I would like them stacked vertically on the left hand side 
with both.  The issue is that, at least on my Samsung Galaxy GT-9000, the 
middle quickslot in portrait mode is so often at the south exit to the screen 
that I end up wasting a potion when I was trying to leave the area.  And I 
would like things on the right, rather than the left, because I don't want to 
occlude the screen with my hand every time I use a potion in battle.
```

Original comment by `160...@gmail.com` on 30 Nov 2011 at 8:45

**Comment by GoogleCodeExporter** [comment]
Original comment by `sdeva...@gmail.com` on 1 Dec 2011 at 5:19
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
With r236, the quickslots will now display the number of remaining items on 
them.
```

Original comment by `oskar.wi...@gmail.com` on 29 Apr 2012 at 4:03
- Changed state: **Ready**
- Added labels: **Milestone-0.6.11**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.11. Closing issue.
```

Original comment by `oskar.wi...@gmail.com` on 9 Jun 2012 at 9:31
- Changed state: **Fixed**
